src/cdk/activation.ts

```diff
@@ -19,9 +18,9 @@ const EXPLORER_ENABLED_CONFIG_KEY = 'aws.cdk.explorer.enabled'
  * Telemetry event names for recorded metrics
  */
 enum TelemetryEventTypes {
-    APP_EXPANDED = 'appExpanded',
-    EXPLORER_RE_ENABLED = 'explorerEnabled',
-    EXPLORER_DISABLED = 'explorerDisabled'
+    APP_EXPANDED = 'cdk.appExpanded',
+    EXPLORER_RE_ENABLED = 'cdk.explorerEnabled',
+    EXPLORER_DISABLED = 'cdk.explorerDisabled'
```

Comment:
Should these be `cdk_` instead of `cdk.`?

Reply:
Yep, nice catch. Fixed